internal/clienttest/clienttest.go

```diff
@@ -47,24 +48,41 @@ func PrivateKey(t *testing.T) *ec.PrivateKey {
 	return key
 }
 
-func GivenSPVWalletClient(t *testing.T) (*client.Client, *httpmock.MockTransport) {
+func GivenSPVWalletClient(t *testing.T) (*client.UserAPI, *httpmock.MockTransport) {
 	t.Helper()
 	transport := httpmock.NewMockTransport()
-	cfg := client.Config{
+	cfg := config.Config{
 		Addr:      TestAPIAddr,
 		Timeout:   5 * time.Second,
 		Transport: transport,
 	}
 
-	spv, err := client.NewWithXPriv(cfg, UserXPriv)
+	spv, err := client.NewUserAPIWithXPriv(cfg, UserXPriv)
```

Comment:
If we changed the name of user and admin client to API and it's package as spvwallet, maybe we also should change the tests package name to something like `spvwallettest`? WDYT

Reply:
I agree, I thought about it while creating the PR, but ultimately I rejected the idea due to the number of files.